docs/INSTALL_es.adoc

```diff
@@ -0,0 +1,133 @@
+:lang: es
+:toc:
+
+= Instrucciones de Instalación
```

Comment:
Should be also "run in place" in spanish
@smoe Creación de LinuxCNC para "ejecutar en el lugar" ?

Reply:
> Should be also "run in place" in spanish @smoe Creación de LinuxCNC para "ejecutar en el lugar" ?

Done

Reply:
Wasn't sure wheather to translate "build" or not. In German it would sound strange if we would translate it.
But doesn't matter at this point, a Spanish speaking person would understand that.

Reply:
Agree that "Run-In-Place" is kind of a proper name.
At least is it referenced everywhere, in any language, as RIP. Which is why, while translating it in "full text" I added the "RIP" mention. But it doesn't really make sense without expliciting it :/